src/main/java/com/github/cameltooling/lsp/internal/completion/CamelOptionValuesCompletionsFuture.java

```diff
@@ -88,6 +89,22 @@ public List<CompletionItem> apply(CamelCatalog camelCatalog) {
 					items.add(langItem);
 				}
 				return items;
+			} else if(StringUtils.startsWithIgnoreCase(
```

Comment:
I think that it can be anywhere in the option, not only in startsWith (but anyway can be done in another iteration as need to check that it is really the case and add more test cases)